proposals/extensions.md

```diff
@@ -601,41 +760,61 @@ The preceding rules mean:
 - and that extension methods declared directly in a namespace take precedence
   over extension methods imported into that same namespace with a using namespace directive.
 
-TODO(static) clarify behavior for extension on `object` or `dynamic` used as `dynamic.M()`?
```

Comment:
:memo: This is converted to a test with a PROTOTYPE comment about diagnostic quality in [PR](https://github.com/dotnet/roslyn/pull/73994)